clients/client-rds-data/protocol/AwsRestJson1_1.ts

```diff
@@ -76,7 +77,8 @@ export async function executeStatementAwsRestJson1_1Deserialize(
   if (output.statusCode !== 200) {
     return executeStatementAwsRestJson1_1DeserializeError(output, context);
   }
-  let data: any = await parseBody(output.body, context);
+  let body = await collectBody(output.body, context);
+  const data = JSON.parse(body);
```

Comment:
This needs to account for a service returning an empty body.

parseBody on this branch is out of date and is doing this currently:
```
const parseBody = (streamBody: any, context: SerdeContext): any => {
  return context.streamCollector(streamBody).then((body: any) => {
    const encoded = context.utf8Encoder(body);
    if (encoded.length) {
      return JSON.parse(encoded);
    }
    return {};
  });
};
```